mlforecast/core.py

```diff
@@ -245,10 +245,17 @@ def __init__(
     ):
         if isinstance(freq, str):
             self.freq = pd.tseries.frequencies.to_offset(freq)
+        elif isinstance(freq, pd._libs.tslibs.offsets.BaseOffset):
```

Comment:
in case they move this to a different module.
```suggestion
        elif isinstance(freq, pd.offsets.BaseOffset):
```

Reply:
fixed